src/hub_predtimechart/__init__.py

```diff
@@ -1 +1 @@
-__version__ = "0.0.1"
+__version__ = "0.0.2"
```

Comment:
Please keep the version aligned with what we plan to release. 

```suggestion
__version__ = "2.2.0"
```

Reply:
Good catch! Done.